docs/content/dagster-cloud/account/managing-users.mdx

```diff
@@ -1,92 +1,128 @@
-
-To add a new user to a deployment:
-
-1. Sign in to your Dagster Cloud account.
+<TODO></TODO>
 
-2. Click the **user menu (your icon) > Cloud Settings**.
-
-3. Fill in the following:
-
-   - **Email** - Enter the user's email address
-   - **Role** - Select the [user role](#understanding-user-permissions) for the user. **Note**: With the exception of the **Organization Admin** role, this role will only apply to the full deployment you're adding the user to.
-
-   For example:
-
-   <Image
-   alt="Cloud Settings Interface for Permissions"
-   src="/images/dagster-cloud/user-token-management/cloud_settings_permissions.png"
-   width={842}
-   height={346}
-   />
+---
 
-4. Click **+ Add**.
+## Understanding user roles
 
-### Removing a user
+<TabGroup>
+  <TabItem name="Overview">
 
-To remove a user from a deployment:
+### User roles
 
-1. Sign in to your Dagster Cloud account.
-2. Click the **user menu (your icon) > Cloud Settings**.
-3. Locate the user in the user list.
-4. Click **Remove**.
-5. When prompted, confirm the removal.
+Dagster Cloud supports six user roles for several levels of role-based access control:
 
-**Note**: This won't remove users from other deployments. For example, if a user has been added to both `prod` and `dev` but only removed in `prod`, they'll still be a user in `dev`.
-
----
-
-## Understanding user permissions
+<table>
+  <thead>
+    <tr>
+      <th width="20%; fixed">Role</th>
+      <th width="15%; fixed">Plan</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <strong>No access</strong>
+      </td>
+      <td>All plans</td>
+      <td>No access to the specified deployment/code location.</td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Viewer</strong>
+      </td>
+      <td>Enterprise</td>
+      <td>
+        The least permissive role. Grants access to view definitions and runs.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Launcher</strong>
+      </td>
+      <td>Enterprise</td>
+      <td>
+        Includes all Viewer permissions and the ability to launch and terminate
+        runs.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Editor</strong>
+      </td>
+      <td>All plans</td>
+      <td>
+        Includes all Launcher permissions and the ability to:
+        <ul>
+          <li>Manage sensors and schedules</li>
+          <li>Manage environent variables</li>
+          <li>Manage code locations</li>
+          <li>Manage agents and alerts</li>
+          <li>View and create users</li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Admin</strong>
+      </td>
+      <td>All plans</td>
+      <td>
+        Includes all Editor permissions and the ability to:
+        <ul>
+          <li>Edit and remove users</li>
+          <li>Manage user tokens</li>
```

Comment:
Any user can view and save their own user token, dunno if we need to call that out